src/chapters/intro-to-data/intro-to-data.rst

```diff
@@ -1,35 +1,34 @@
 Introduction To Data
 ====================
 
-Read the following articles, follow along where instructed:
-
 Methods and Techniques
 ----------------------
 
-1. `Data Analysis Methods & Techniques <https://www.datapine.com/blog/data-analysis-methods-and-techniques/>`__.
+Read the following article.
```

Comment:
To conform with later chapters please add Read the following articles., above Methods and Techniques.

Delete all other instances on the page.